### ISSUE TITLE
Hyper-parameters selection

### ISSUE THREAD

**Issue by Looomo** [jannerm/diffuser#29]
Hi, 
    Thanks for the wonderful work! Just for confirmation，which group of parameters should be used for training and planning?  The parameters in the loaded pretrained weight seem different from the default parameters in the scripts and the parameters listed in the last page of your paper. For example, your paper suggests 100 diffusion steps while it is only 20 in your pretrained weight. 
    Thanks!

**Comment by jannerm** [comment]
The default hyperparameters in the repo should be consistent with [the current version on arxiv](https://arxiv.org/abs/2205.09991) (v2 at the time of writing) and reproduce the numbers in the tables. Let me know if there's a remaining discrepancy.

(The reason why the defaults are different between v1 and v2 is because we re-tuned planning to be much faster in terms of wall-clock time while retaining the original performance.)

**Comment by Looomo** [comment]
Thanks, that makes sense~

**Comment by Looomo** [comment]
@jannerm Hello again, I have checked the vaiables of walker2d-medium-replay-v2, which shows that the model is trained 780,000 steps with lr 0.002, which are 500k and 4e-5 in the paper. I believe that should be 780,000 steps with lr 0.002 right? Cause I didnt make it to reproduce the results by training 500k with lr 4e-5(on a RTX3090). Expirement of training 780,000 steps with lr 0.002 is still running.

**Comment by jannerm** [comment]
Did you mean `lr = 0.0002` instead of `0.002`? If so: nice catch! I will update that in the arXiv v3.

Just to clarify, where did you see 780,000 training updates?

**Comment by Looomo** [comment]
Sorry for my typo, I mean `lr = 0.0002` instead of `0.002`.
As for the 780,000 training steps, I saw it in the pretrained model, as below. The loaded weight is named as `state_600000.pt`, I believe that  is because the label frequency has not been reached， which is set to 200k.

Also, other datasets have different training steps too. And it seems impossible to load a 500k model form the pretrained weights since `label_freq` and `save_freq` are set to `200,000` and `20,000`, which means `state_400000.pt` is actually trained 580,000 steps.

![image](https://user-images.githubusercontent.com/29702226/213895226-794c47cc-4326-40c5-a3a4-d7e0d369aa6e.png)

**Comment by jannerm** [comment]
Yeah, it's not important that the filename matches -- the epoch labels will be rounded down to the nearest `label_freq`, so won't match the highest train step in general.

The trainer showing 780,000 training steps is more unexpected. My first thought was that this model was trained in an older version of the repo in which the number of training steps was scaled by the number of samples in the dataset; `walker2d-medium-replay` is smaller than the average dataset, so would have fewer than the default 1 million steps. But I save the commit and diff for all experiments, and after checking out the state of the codebase from that experiment and [applying the relevant patch](https://drive.google.com/file/d/1UI8QlggpvMkdSSX22tvT7I1ceAWuheqb/view?usp=sharing), it does not appear that this experiment was run with that feature.

Since it wasn't intentional training length scaling, that suggests that the job was terminated early on the cluster, 78% of the way through training, and I didn't notice. What that means is that the model is slightly undertrained compared to the intended number of training steps in the config file.

**Comment by Looomo** [comment]
Hi there, problem solved. I have trained diffuser on `walker2d-medium-reolay-v2`  on 3090 and A6000, and evaluate the trained weight for 40 times, which have results around `66±4`. I'll take the differences between `70.6` and `66` as performance drop due to hardware differences.

Thanks for your reply these days!

**Comment by jannerm** [comment]
Great, thanks for letting me know! Did you have to change anything between five days ago and today?

**Comment by Looomo** [comment]
Nothing changed actually. Training `1,000,000` or `780,000` steps with lr `0.0002` and batchsize `32` have the similar results for me (both `66±4`). By the way, training with `3200` batchsize to speed up the training process seems leads to worse results (around `45` on `walker2d-medium-replay-v2`), just for supplement.

**Comment by jannerm** [comment]
That's good to know, thanks for the update!
